http-gateway/pkg/server/gateway_integration_test.go

```diff
@@ -143,18 +142,11 @@ func post(port int, path string, body io.Reader, headerKV ...string) *http.Respo
 	return resp
 }
 
-func waitForHttpGatewayToBeReady() {
-	attempts := 20
-	durationBetweenAttempts := time.Millisecond * 100
+func waitForHttpGatewayToBeReady(port int) {
+	url := fmt.Sprintf("http://localhost:%d", port)
 
-	for i := 0; i < attempts; i++ {
-		_, err := net.Dial("tcp", ":http")
-		if err != nil {
-			fmt.Print(".")
-			time.Sleep(durationBetweenAttempts)
-			continue
-		}
-
-		break
-	}
+	Eventually(func() error {
+		_, err := http.Get(url)
+		return err
+	}).Should(Succeed())
```

Comment:
`Eventually` is taking the default timeout of 1 second. I know this seems like aeons, but on a really slow (CI) system, it might not be enough. How about upping it to 10 seconds?

Reply:
Ah, good call.